### PR TITLE
Add Extrada data to Jitsu events / connect all with jitsu

### LINKED ISSUE #23526: We need to register extra data in each jitsu event
### Is your feature request related to a problem? Please describe.
We need to register some extra values like: experiment's id, variant's name use by the user and current lookBackWindows values, language and persona, this values are going to be use later when we gather the data from cubejs.

### Describe the solution you'd like
Some extra values like lookbackWindows, experiment's id, variant's name need to be set in the bowser because we have all this values into the local storage, I think the best way to do this is using this: https://jitsu.com/docs/sending-data/js-sdk/snippet#persistent-properties 

In the case of language and persona we need to set these into the server, right here:

https://github.com/dotCMS/core/blob/0addf9cd0e8e0951b628185c17e3417c07d129ad/dotCMS/src/main/java/com/dotcms/jitsu/EventLogWebInterceptor.java#L107

### Acceptance criteria
- lookbackWindows, experiments'id and variant's name should be register in any jitsu event

### PR DESCRIPTION
**Pull request by freddyucv**
### Proposed Changes
* Add new fields into the Clickhouse Events table 

https://github.com/dotCMS/core/pull/23698/files#diff-81c2b7651a61e07b6120a6b74f75a426dfda5d335f4d762b5d7bb83d4f6ba004R46

https://github.com/dotCMS/internal-infrastructure/pull/149/files

* Remove the code to create cookies, we are not going to need the Cookies after all

https://github.com/dotCMS/core/pull/23698/files#diff-df4c3ee11b473e0f4e91d5be7a6107bf6cdb4ab40bf0661a182dd298e00e5602L75

* Create a Builder for SelecetedExperiment

https://github.com/dotCMS/core/pull/23698/files#diff-282d55898ddcc13745c364b2a7277eea0babbd5ab656e4100748e1a886ece800R79

* Send A Event for each Running Experiment

https://github.com/dotCMS/core/pull/23698/files#diff-c6b552720dcdd032d58c9b299fd4c5aba7a5f3b233c6ae1d7a1528e94bfe51c7R63

* for now I am commented this line

https://github.com/dotCMS/core/pull/23698/files#diff-d728800ddc8741a2057c0709a77ccf4c9d2e6c833536579add9b3ba8b922b9ddR124

When try to save the clusterId we are getting a error that the Events Table in ClickHouse is not allow to be alter, also we are already saving a clusterId with the value of 'cluster1' so I don't if it is ok.

### CODE REVIEW

**Review by jdotcms — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/jitsu/EventLogWebInterceptor.java [hunk 1]
```diff
@@ -98,18 +101,36 @@ public void doPost(final HttpServletRequest request,
...
                 ? realIp.substring(0, realIp.lastIndexOf(".")) + ".1"
                 : "0.0.0.0";
-        json.put("ip", mungedIp);
-        json.put("clusterId", ClusterFactory.getClusterId());
+        eventPayload.put("ip", mungedIp);
+        eventPayload.put("userLanguage", String.valueOf(
+                WebAPILocator.getLanguageWebAPI().getLanguage(request).getId()));
+        eventPayload.put("persona",
+                WebAPILocator.getPersonalizationWebAPI().getContainerPersonalization(request));
+        //eventPayload.put("clusterId", ClusterFactory.getClusterId());
```
Comment (freddyucv) [inline dotCMS/src/main/java/com/dotcms/jitsu/EventLogWebInterceptor.java:124]: For some reasons when we set this clusterID we are getting an error in jitsu.
The error something like that it is not be able to ALTER the events table in clickhouse to add a new field call clusterid.

Also the events already has a cluster_id set, I don't know were we are setting this and if this is ok

Comment (freddyucv) [reply dotCMS/src/main/java/com/dotcms/jitsu/EventLogWebInterceptor.java:124]: I am going to create a new issue to fix (if it is needed to fix) all this later

Comment (freddyucv) [reply dotCMS/src/main/java/com/dotcms/jitsu/EventLogWebInterceptor.java:124]: Issue created https://github.com/dotCMS/core/issues/23834

**Review by dsilvam — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/jitsu/EventsPayload.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            this.jsonObject = jsonObject;
+        }
+
+        @Override
+        public String toString() {
+            return jsonObject.toString();
+        }
+    }
+
+    private static class ShortExperiment{
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotcms/jitsu/EventsPayload.java:62]: rename to LiteExperiment

Comment (freddyucv) [reply dotCMS/src/main/java/com/dotcms/jitsu/EventsPayload.java]: donehttps://github.com/dotCMS/core/pull/23698/commits/c9ba2236c8443e19a8f0a7b2d82e51a8ec94d26c

File: dotCMS/src/main/java/com/dotcms/jitsu/EventLogWebInterceptor.java (hunk 1 above)
Comment (wezell) [reply dotCMS/src/main/java/com/dotcms/jitsu/EventLogWebInterceptor.java:124]: We have to have the cluster_id set in clickhouse. This is how we multi-tenant

**Pull request merged**